src/modules/physics/physics.c

```diff
@@ -517,10 +517,7 @@ void lovrColliderGetOrientation(Collider* collider, float* angle, float* x, floa
   quat_getAngleAxis(quaternion, angle, x, y, z);
 }
 
-void lovrColliderSetOrientation(Collider* collider, float angle, float x, float y, float z) {
-  float quaternion[4];
-  quat_fromAngleAxis(quaternion, angle, x, y, z);
-  float q[4] = { quaternion[3], quaternion[0], quaternion[1], quaternion[2] };
```

Comment:
I think we still need to swizzle the quaternion components here (unless it's working as is)

Reply:
Ouch. Yeah, that reordering is still needed.